style.css

```diff
@@ -0,0 +1,71 @@
+body {
+  background-color: rgb(141, 107, 56);
+}
+
+.classbody {
+  background-color: rgb(211, 162, 82);
+  width: 432px;
+  margin-left: 500px;
+}
+
+h2 {
+  font-family: "Anton", sans-serif;
+  margin-left: 50px;
```

Comment:
All throughout your CSS, you need to repeat this left margin. It would be simpler and easier to keep up if you instead put a wrapper `div` around all the content after your `img` on line 16 in your HTML and gave that a class name and some padding -- then everything inside that `div` would be pushed away from the edges of the card, [like so](https://glitch.com/edit/#!/remix-natan-p2).